Core1Platform/UserInterface/01a_Profile.md

```diff
@@ -0,0 +1,38 @@
+[!!Edit profile settings](../UsingCore1/02_EditProfileSettings.md)
+
+# Profile
+
+*Any workspace > Avator icon > Menu entry Settings > Tab PROFILE*
+
+![Profile](../../Assets/Screenshots/Core1Platform/ProfileSettings/Profile/Profile.png "[Profile]")
+
+**Edit user**
+
+- [SAVE]   
+    Click this button to save any changes. The notice *Submitting data...* is displayed. When the changes are saved, the *User saved* pop-up window is displayed.
+
+    ![User saved](../../Assets/Screenshots/Core1Platform/ProfileSettings/Profile/UserSaved.png "[User saved]")
+
+- *Login*   
+    Enter the user name.
+
+- *First name*   
+    Enter the first name of the user.
+
+- *Last name*   
+    Enter the last name of the user.
+
+- *Email*   
+    Enter the email address of the user.
+
+- *Address*   
+    Enter the street and house number of the user's address.
+
+- *ZIP*   
+    Enter the ZIP code of the user's address.
```

Comment:
Wir haben "Zip" (Großschreibung/Satzanfang) und "zip code" verwendet, da im Wörterbuch so als üblicher angegeben wurde (s. Link unten). Sicher war es am Anfang ein Acronym, genauso wie laser und andere Wörter, aber ich würde sagen, jetzt kann man als normales Wort klein schreiben. Das ist umstritten (viele Kommentare dazu in Internet), aber lass uns konsistent bleiben.

https://www.merriam-webster.com/dictionary/zip%20code